go/master/etcd_client.go

```diff
@@ -18,6 +18,19 @@ const (
 	DefaultAddrPath = "/master/addr"
 )
 
+// DatabaseOperator is an interface fo database operator,
+// it's useful for unittest
+type DatabaseOperator interface {
```

Comment:
Please see my other comment on "removing `NewEtcdMasterClient`". Maybe we don't need this interface anymore.